playground/frontend/lib/modules/sdk/components/sdk_selector.dart

```diff
@@ -67,15 +67,7 @@ class SDKSelector extends StatelessWidget {
                     onSelect: () {
                       close();
                       setSdk(value);
-                      setExample(
-                        state.defaultExamplesMap![value] ??
-                            ExampleModel(
-                              name: kEmptyExampleName,
-                              path: '',
-                              description: '',
-                              type: ExampleType.example,
-                            ),
-                      );
+                      setExample(state.defaultExamplesMap![value]!);
```

Comment:
These exclamation marks are a code smell. Can they be fixed by added asserts or default initializing?

Reply:
Hello @KonradJanica! Thank you for the review! I'll fix this in the next commit.